test/models/classroom_test.rb

```diff
@@ -45,4 +45,47 @@ class ClassroomTest < ActiveSupport::TestCase
     assert_includes teacher.classrooms, classroom1
     assert_includes teacher.classrooms, classroom2
   end
+
+  test "has many students through classrooms association" do
+    classroom = create(:classroom)
+    student1 = create(:student, classroom: classroom)
+    student2 = create(:student, classroom: classroom)
+    
+    assert_includes classroom.students, student1
+    assert_includes classroom.students, student2
+    assert_equal 2, classroom.students.count
+  end
+  
+  test "has many teachers through teacher_classrooms association" do
+    classroom = create(:classroom)
+    teacher1 = create(:teacher)
```

Comment:
Each of the teacher creations here will create a classroom then you're adding them to a different classroom.  I'm not sure it's necessary for the teachers to belong to two classrooms for this test.  Maybe we can just create the teacher with the classroom since you've already created it, like you did on line 51.

Reply:
Good callout! I've updated this so that we're more closely mirroring the object setup in the student test, but doing it in such a way that appropriately sets up the `teacher_classroom` association.

I've also updated the linting in the test file 😊